### PR TITLE
add practice-linux-commands-free-course to courses/free-courses-en.md

### PR DESCRIPTION
**Pull request by huhuhang**
## What does this PR do?
Add resource(s)

### Description

LabEx provides free Linux command practice tutorials and an online virtual environment. The hands-on approach is very suitable for beginners to get started with the tutorials.

### Why is this valuable (or not)?

Practice Linux Commands (Hands-On Labs)" has been learned for free by hundreds of thousands of users, with over 98% positive reviews

### How do we know it's really free?

Free access without registration: https://labex.io/tutorials/practice-linux-commands-free-tutorials-398420

Here are example chapters:

1. The first chapter:  https://labex.io/tutorials/linux-linux-ls-command-content-listing-219205
2. The last chapter: https://labex.io/tutorials/linux-linux-time-command-command-timing-219197

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1265,6 +1265,7 @@
 * [Linux Tutorials and Projects](https://www.udemy.com/course/linux-tutorials/) - Jason Cannon (Udemy)
 * [LPIC 1 exam guide: Linux Professional Institute Certification Study Guide](https://linux1st.com) - Jadi Mirmirani (HTML)
 * [LPIC 2 exam guide: Linux Professional Institute Certification Study Guide](https://borosan.gitbook.io/lpic2-exam-guide) - Payam Borosan (HTML)
+* [Practice Linux Commands (Hands-On Labs)](https://labex.io/tutorials/practice-linux-commands-free-tutorials-398420) - LabEx
```
Comment (eshellman) [inline courses/free-courses-en.md:1268]: I think this belongs in our interactive tutorials list. Could you take a look at that list and see if you agree?

Comment (huhuhang) [reply courses/free-courses-en.md:1268]: > I think this belongs in our interactive tutorials list. Could you take a look at that list and see if you agree?

Thank you for your suggestion! I wasn't aware of the interactive tutorials list, and it is indeed a better fit. I'll relocate it to more/free-programming-interactive-tutorials-en.md. 😄

**Comment by huhuhang** [comment]
@eshellman Hello, the modifications are complete. Thank you for your reminder and review.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**